clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/model/Cluster.java

```diff
@@ -69,7 +69,7 @@ default Moniker getMoniker() {
    */
   @Empty
   @JsonSerialize(nullsUsing = NullCollectionSerializer.class)
-  Set<ServerGroup> getServerGroups();
+  Set<? extends ServerGroup> getServerGroups();
```

Comment:
Allows us to use Lombok `@Getter` on model implementations while holding a more specific implementation of `ServerGroup`, e.g. `CloudFoundryServerGroup`. Much less casting when using the model object later.

Due to Groovy's relaxed type system, this likely never came up before.